assets/MacroAPI.svg

```diff
@@ -0,0 +1,40 @@
230ENBe/9yDk47CRZvBdFRhbtPL3RwyIE3fVC514+Cy3F8Hz99ddbesr31kB6ynyKnhEe7Zz7D2scNYsOueM9OntyjRCdhHIj/CH6U+W9JlU7BshvXde9vri4eCHzmWArnl6LQjg4HKQVvSAh8RgeCFpaTylzusUitrxEWUrnRbiEIYUxkICtwnMUcplAFQQxRB999FH68MMPdwKkip23QWEmjdJIljFmOkaRVHIMHv+v9BRm/PLLL3cMmcBz6We4CU/a/c9//rOl5/3797evu3fvXot0+SY6C1bE7Ql9Tgl4c4S/a3+EbiLYYsxlHDIe4VHe6oQVVw1dI57Ra0s/ZD712sKbQk9RQryqLIGMW0qI6d9TfYbpyvCum2B+lDZYK1OeE1lT50TlnXlxSuNuZQlERpSeIvNCT3WSeC5Zd6EOyPWTj2MDwzq5g922c0ZKxyPHC3+K/lR5lzFNFYlbUId2cCovLi4uXv7iF794dyOq5ONkc7jhJqFrhqWEwXLHWJ/ZG9vQ5p44ISLQGvkI0eQln+W31vcGMDRV1UGRNA2pJImGOAKxBBWF2PLWLZrmvFI8Riad6ajKR3+TdkWgRbjFCAlNZVythDsCRmIi3PJZDbpGQ6qkZGxsoNEjtTxUj0/RqJegp90N9DxVlsKnMgahq9LScgDmoKs6SMpnyBMJokmhKaaXcnzI/Oyl1q3/vFqK0o8jD6Gh8KXSVdOJLeiHY2LHUXRNB/cjiUGXyCIZe1da9EqB/PJYIl1qHcM6k3Wq0E2/qw6VCFl4VPWn1Y+pMcz9xZs3b775/e9/3/3yl7/c3vsjllDuRH1zdna2UeHFlSee4Ob2abImgyMdLOJjgYy9bq0TaIFVPU3rPoE5ipS8EalMpkY/umSUjYzSFGlm5ZU9enqekQK9HVx9hd4k3mMj/dBQXFObniL2vNipIO1qOlUFRAQc7/9Jxkakuk2LxYss4B4f1xZi1QgiLVVZSt/VMVIjeohbBbohuhVPV5S3vNA5QsepH1Lcmv7CazDdE9RDrOyIx7OWbCNd8CZT/F9pKrwh7/K9Nd0sSF/UCEo/NAWvtLD4Mr3z+nf/RQYKHW5PZybIBHTGoqweVrelod9IN9WfQk95z0VQU9J0mFtRnK9/9atfvasBffHFF/L2RlJxrCCRsExkT7C9VWGJlAEvoe5ptYleB4mMAs9pEJ6UqcE5dFXyqmjQG8exar8sBanMWmuAGOz14HJq/Y8FXOcKc+qstPdd/VMCbUuVkNb3OJLRG1KxNoS0TeSNWh7mFAYIjbmlMNWTx9TbnBE69lMVtyhMNNSqNLVvugefZ4C8qMjz1lkOUUFaBogjRZZ5rPO2hBelap+0viY0VYcS6ytME65lMl3TdQUdGiCmEdOLHfcE+hN1JzrpqINaOO+YNt9sNq/6vn/98OHD7uHDh28jIAlx8Sl1jI7uQI48TBxEZFU9wfcE1TJu+F8kDFNAacATjEtrmZG8MVh95/+tz2yAmMnw8xivm+fZSoe2AD9SAD05FojcCqiIbt7nWofFMsqeczB3ehjRQe5fVxkmuqlY6Y1RTQpolgxP3ZsPVmjMozk5OQTdPAPMPKKrRXNyb918bPFlCgyQ1T/Vnd7cYFviWCRIy/P/aCBaZY8GZ0d2QbjcbDYvf/Ob37yLgAZcegYo1zE2QC0YJ2eYlEG6hqmiiCkw/cX9wuOm7MOU4+wgnYXKgRc8tJ5bjrR7KpAfUqFHsBR3a8OdQ+ekpC3n5BBpQg8ezciTnrxdy8nFrMWG7kVLlD6z+t2arjn+4swHgsssrXhVI8ehD6/RzugybPlj0/f9KPPnWXTuRM21+DcUcC8Mba0kvYm0+m2NozQq8s6LtlqpGTPOlxeu830jLVNJUcTMv/ES/X1g8VQOvArP8ihL+hdF9FNA02qa/sFcP877GOXDdONoKUrB8/kJZNfKrDAftgTTQWUeU2jYZ4xurKxELayIOrpmDT2YhlyyaHlDKsiKNH5xzQANDW9XweUGkGsoNWCUKCydOsKIEBXomXmt8LhEwXrXz/2eC8Wt/yxh887lJeVTwlOCOWO/L5+NOf+MnjiKNQzuszeu1ooUYaUK0aHQyHeMc2Gl4FQx1+oCTRVFTuVcxoidXH7asfaRo/Ep55mNM7cRpekYlpOAjoLnTE2N4Xafi+G1he6EIH9cjo2A5kgzRMp7rjSHp0xy+doIU/W9xgB5KDFaLYCpP27Tis4OmdbCBTDaj8gxUViKo+U4MHrglU5WDWtMWquntHMyxlQ7Rlbs6KXP4WBa8hy1HdFsqmgod0zJscyL7Gy2TsNB+99ci4CGMHIzvK6gxOPYR3Ht68lY9yC0UFJYm+AUg6VQcuOaun/s5UT9yM2l1beWwu+F/5bSruHDmjRojTcZrcjivs6VPvL6glBlbslH59zPM1Z+S85DOkWGhlcTtuJFnCurtmPJOvMO87F3j1QNXa0x1+qa0vlosQiB2n5xzQANB8gorvUyEiBvUDUM0uLYFgzKqQGrvTnTgRamMBzeNVreW5UTEE7F1Fx3yuNKrol95fTc2P6NPc9aYWpFPtHCjqkjaT6mo9WPzGc9LS5qHQlFir5k7kt+K7lWzbFT0KO1vhquLwz5FAMdNEA9vK5Zm7EFqqkigdqJbFGHirx0brd15GfB8orGGKUonXcIT75U6ZSkIT1HqkYAc1FYTlb2pWetzLBBTMQf+xSfx8pfZEy8dNEYjKWvJ0tMU+9YvNYYlETzpW2U0EDH0Vi+1QDtOowpuD4ZEVBrHEKhjUFtP5eQcpmqH4eeo7Ht1xTAp8IctJqSHoea20PMTSm8NsfQb8n8MPU1CiAG6OtrBgg8oM6KflasWLFixYo9IbWfr/ASZ/C+Gp4VK1asWNEKL1NKjx4+fHi1BvT+++/3a/SzYsWKFSsa4tWQgttha4A++eQTiYDOISJasWLFihUrpoTcA/TOAKWU/g8jW9n9EWo8PQAAAABJRU5ErkJggg=="/>
```

Comment:
Is it possible to use vector graphics instead of an embedded png?

Reply:
I don't think so as it uses images for the keycap and background of the keyboard, It would be quite difficult to replicate it using just hues and shapes.

Reply:
That's fine, a png is perfectly acceptable.